target/linux/ramips/dts/YOUKU-YK2.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "youku,yk2", "mediatek,mt7621-soc";
```

Comment:
I believe YK-L2 is the actual model number, so you can set it to youku,yk-l2. You don't need to follow the previous youku-yk1 mistake.

Reply:
Yes , `YK-L2` is the actual device name.
I set it to `youku,yk-l2` , but looks everything is not necessary set to `L2`.